### ISSUE TITLE
Session issue

### ISSUE THREAD

**Issue by anurajpv** [haseydesign/flexi-auth#37]
After logout, and press back button of your browser, It shows logged-in.

**Comment by nathan815** [comment]
I don't think this is an issue. Have you tried reloading the page after pressing the back button? The browser is probably just displaying from memory/cache to load faster so it appears to still be logged in.

**Comment by ishtiaq12** [comment]
hello guys the problem i m facing with flexiauth is when i open two different applications on my local server and both applications using same flexiauth library for authentication , The problem is when i logged in one application the other is automatically logged out ... i cant understand why ... bcz both applications have their own separated databases... also different session names are set mentioned in config file... can any one help me out .....it will be highly appreciated ... thanks

**Comment by rmohit05** [comment]
$parent_user_id = $this->flexi_auth->get_user_id() i am empty value after login flexi auth in Contrller B
